sbi/neural_nets/embedding_nets.py

```diff
@@ -31,3 +31,136 @@ def forward(self, x: Tensor) -> Tensor:
         """
         x = self.net(x)
         return x
+
+
+class CNNEmbedding(nn.Module):
+    def __init__(
+        self, input_dim: int, output_dim: int, num_fc: int = 2, num_hiddens: int = 120
+    ):
+        """multi-layer (C)NN
+            first 2 layers are convolutional, followed by fully connected layers.
```

Comment:
Would be cool if the number of convolutional layers would be parameter instead of fixing it to `2`